includes/components/class-attribute.php

```diff
@@ -1721,6 +1725,68 @@ public function add_meta_boxes( $meta_boxes ) {
 				// Add meta box.
 				$meta_boxes[ $model . '_' . $meta_box_name ] = $meta_box;
 			}
+
+			// Set model category settings key.
+			$category_key = $model . '_category_settings';
+
+			if ( ! $this->models[ $model ]['searchable'] || ! in_array( $category_key, array_keys( $meta_boxes ), true ) ) {
+				continue;
+			}
+
+			// Create sort form.
+			$sort_form = hp\create_class_instance( '\HivePress\Forms\\' . $model . '_sort' );
+
+			// Check sort form.
+			if ( ! $sort_form ) {
+				continue;
+			}
+
+			// Get sort options.
+			$model_options = $sort_form->get_fields()['_sort']->get_arg( 'options' );
+
+			// Add category options.
+			$category_options = [];
+
+			if ( isset( $_REQUEST['tag_ID'] ) ) {
```

Comment:
Why this check is needed?

Reply:
to check if user is on creating or editing category page